src/Admin/MoocAdmin.php

```diff
@@ -158,15 +152,23 @@ protected function configureListFields(ListMapper $listMapper): void
         ;
     }
 
-    protected function configureRoutes(RouteCollection $collection): void
+    private function addImageSrc(): ?string
+    {
+        return $this->getSubject() && $this->getSubject()->getImageName()
+            ? '<br/>><img src="'.$this->getSubject()->getAssetImagePath().'" />'
```

Comment:
There is a useless `>` char : 
```php 
<br/>><img src
``` 
=>
```php
<br/><img src
```
What about using a method (`getImageHelpMessage()`) to directly return the image help message with the img tag if an image is uploaded  ?